tests/system/action/gender/test_create.py

```diff
@@ -0,0 +1,116 @@
+from typing import Any
+
+from openslides_backend.permissions.management_levels import OrganizationManagementLevel
+from openslides_backend.shared.util import ONE_ORGANIZATION_FQID
+from tests.system.action.base import BaseActionTestCase
+
+
+class GenderCreateActionTest(BaseActionTestCase):
+    def setUp(self) -> None:
+        super().setUp()
+        self.test_models: dict[str, dict[str, Any]] = {
+            ONE_ORGANIZATION_FQID: {"name": "test_organization1"},
+            "user/20": {"username": "test_user20"},
+        }
+
+    def test_create(self) -> None:
+        self.set_models(self.test_models)
+        self.set_models({"gender/1": {"organization_id": 1, "name": "male"}})
```

Comment:
You are using the test_models only once. Then please join organization and user setting with the gender/1 setting in line 18 and remove it from setUp-method.
Additionally you should add the opposite relation (field organization/gender_ids) to the organization/1-record and assert the gender/1 and gender/2 instances in organization.

Reply:
Done